test/integration/multipleDepositWithdraw.js

```diff
@@ -0,0 +1,326 @@
+     originUpdateNonce = logs[0].args.leafIndex;
+     originMerkleRoot = await OriginChainLinkableAnchorInstance.getLastRoot();
+     // create correct update proposal data for the deposit on origin chain
+     originDepositData = Helpers.createUpdateProposalData(originChainID, originBlockHeight + 10, originMerkleRoot);
+     originDepositDataHash = Ethers.utils.keccak256(DestAnchorHandlerInstance.address + originDepositData.substr(2));
+
+    // a second deposit on origin chain leads to update edge proposal on dest chain
+    // relayer1 creates the deposit proposal for the deposit that occured in the before each loop
+    TruffleAssert.passes(await DestBridgeInstance.voteProposal(
+      originChainID,
+      originUpdateNonce,
+      resourceID,
+      originDepositDataHash,
+      { from: relayer1Address }
+    ));
+
+    // relayer2 votes in favor of the update proposal
+    // because the relayerThreshold is 2, the deposit proposal will become passed
+    TruffleAssert.passes(await DestBridgeInstance.voteProposal(
+      originChainID,
+      originUpdateNonce,
+      resourceID,
+      originDepositDataHash,
+      { from: relayer2Address }
+    ));
+
+    // relayer1 will execute the deposit proposal
+    TruffleAssert.passes(await DestBridgeInstance.executeProposal(
+      originChainID,
+      originUpdateNonce,
+      originDepositData,
+      resourceID,
+      { from: relayer1Address }
+    ));
+    // check roots
+    const destNeighborRoots = await DestChainLinkableAnchorInstance.getLatestNeighborRoots();
+    assert.strictEqual(destNeighborRoots.length, maxRoots);
+    assert.strictEqual(destNeighborRoots[0], originMerkleRoot);
+
+    // check initial balances
+    let balanceOperatorBefore = await destChainToken.balanceOf(operator);
+    let balanceReceiverBefore = await destChainToken.balanceOf(Helpers.toFixedHex(recipient, 20));
+    let balanceDestAnchorAfterDeposits = await destChainToken.balanceOf(DestChainLinkableAnchorInstance.address);
+    // insert two commitments into the tree
+    await tree.insert(Helpers.generateDeposit(destChainID).commitment);
+    await tree.insert(originDeposit.commitment);
+  
+    const { root, path_elements, path_index } = await tree.path(1);
+    
+    // verification fails (this discrepency is likley one of the reasons)
+    assert.strictEqual(destNeighborRoots[0], Helpers.toFixedHex(root))
+
+    const destNativeRoot = await DestChainLinkableAnchorInstance.getLastRoot();
+    const input = {
+      // public
+      nullifierHash: originDeposit.nullifierHash,
+      recipient,
+      relayer: operator,
+      fee,
+      refund,
+      chainID: originDeposit.chainID,
+      roots: [destNativeRoot, destNeighborRoots],
+      // private
+      nullifier: originDeposit.nullifier,
+      secret: originDeposit.secret,
+      pathElements: path_elements,
+      pathIndices: path_index,
+      diffs: [destNativeRoot, destNeighborRoots].map(r => {
```

Comment:
```suggestion
      diffs: [destNativeRoot, ...destNeighborRoots].map(r => {
```
This looks incorrect. `destNeighborRoots` is an array containing 1 root, yet you haven't passed it in as that individual root.